presto-main/src/main/java/com/facebook/presto/execution/QueryExecution.java

```diff
@@ -53,6 +53,11 @@
 
     QueryInfo getQueryInfo();
 
+    default DataSize getScannedBytes()
+    {
+        return new DataSize(0, DataSize.Unit.BYTE);
+    }
+
```

Comment:
The thought here is that we are implementing limit for scanned bytes from raw input's perspective, eventually we can add more knobs to control the scanned bytes for output data size, post-compressed size, etc. the default scannedBytes gives a general guideline for all implementations with datasize=0b to start with

Reply:
My apologies - your point is not very clear to me. `scannedBytes for output data size` - should we just not call it `getOutputDataBytes`. I am not sure what kind of guidance are we offering when we start with `datasize=0b`

Reply:
I was reading through this again and have the same question.  Why make this a default, why not just move this default method to `DataDefinitionExecution`?

Reply:
Sorry, missed this comment. yes agree with you all, so instead of going to default value, I will just remove this definition and simply take session value and config value into account. If we want to implement further based on scannedbyte config like scan limit for different connectors, we can add it back to set a bar as a default scan byte limit for general purpose, but for now I will remove this block of code